### PR TITLE
Modéliser la réforme des AL 2019

### LINKED ISSUE #14: Modéliser la réforme des AL 2019
Relates to openfisca/openfisca-france#888

Dans le cadre du GT non-recours, nous allons modéliser des réformes pour connaître l'impact des propositions du groupe sur des situations types.

### PR DESCRIPTION
**Pull request by sandcha**
Fixes #14 

* Déplace les exemples python tel que le code du GT dans `tutorial/exemples`
  * Ajoute la dernière version du code python & R du GT non recours dans `tutorial/exemples/gt_non_recours`
* Conserve le code du framework des formations dans `tutorial/python`

### CODE REVIEW

File: exemples/gt_non_recours/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  
+ > Supports du GT `Accès aux droits et aux services, lutte contre le non recours`  
+ de la `Délégation Interministérielle à la prévention et à la lutte contre la pauvreté des enfants et des jeunes`  
+ Le rapport de ce Groupe de Travail est accessible [ici](https://www.caissedesdepotsdesterritoires.fr/cs/BlobServer?blobkey=id&blobnocache=true&blobwhere=1250171076233&blobheader=application%2Fpdf&blobcol=urldata&blobtable=MungoBlobs).
+
+### Description du contenu
+
+* Le script `framework_no_reform/investiguer_achetypes.py` execute un ensemble de calculs sur une ou plusieurs periodes et sur les situations contenues dans `/situations`.  
+Les resultats sont repertoriés dans un fichier `csv`.
+
```
Comment (Anna-Livia) [inline exemples/gt_non_recours/README.md]: This file seems to not be in this directory anymore ...

Comment (Anna-Livia) [reply exemples/gt_non_recours/README.md]: investiguer_achetypes.py --> investiguer_archetypes.py

File: exemples/gt_non_recours/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* Le script `framework.py` exécute un ensemble de calculs de variables OpenFisca, d'après un ensemble de réformes sur les situations contenues dans `/situations`.
```
Comment (Anna-Livia) [inline exemples/gt_non_recours/README.md]: I would add the install and run procedure here.(`make install` / `make run`)

Comment (Anna-Livia) [inline exemples/gt_non_recours/README.md]: And maybe check some of the requirements such as `python 2.7`

[the remark this answers is not in the document]
Comment (sandcha) [reply exemples/gt_non_recours/README.md]: Line removed as `framework_no_reform/investiguer_achetypes.py` wasn't used by GT analysis. But, text kept in python new README file.

File: exemples/gt_non_recours/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* Le script `framework.py` exécute un ensemble de calculs de variables OpenFisca, d'après un ensemble de réformes sur les situations contenues dans `/situations`.  
+Les resultats sont repertoriés dans un fichier `csv`.
+
+* Le code `vizu.R` en langage [R](https://www.r-project.org) génère des courbes à partir des résultats `resultats-levels-.csv` du `framework.py`.
+
+### Installation & exécution de `vizu.R`
+
+Les résultats OpenFisca présentés au GT comprennent des graphiques générés à partir de `vizu.R`. Celui-ci nécessite un environnement spécifique au langage [R](https://www.r-project.org) pour être exécuté :
+
+1. Installer la version 3+ du langage R :
```
[the remark this answers is not in the document]
Comment (sandcha) [reply exemples/gt_non_recours/README.md]: It depends on what we aim to do by opening this code. As I see it, it's a real example using openfisca in a specific context and its main purpose is to allow a replication of the context and our published results for this GT. It shouldn't be updated in this case but `tutorial` might not be its final repository. 🤔

File: exemples/gt_non_recours/README.md (hunk 1 above)
Comment (sandcha) [reply exemples/gt_non_recours/README.md]: Moved into new `python/README.md`.

File: exemples/gt_non_recours/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    year           2016
+    month          06
+    day            21
+    svn rev        70800
+    language       R
+    version.string R version 3.3.1 (2016-06-21)
+    nickname       Bug in Your Hair
+    ```
+
+4. Dans RStudio, spécifier le répertoire de travail à partir duquel les chemins de fichiers seront calculés : dans le menu `Session`, choisir `Set Working Directory` > `Choose Directory...` et indiquer le répertoire `./exemples/gt_non_recours` où se trouve le code R `vizu.R`.
```
[the remark this answers is not in the document]
Comment (sandcha) [reply exemples/gt_non_recours/README.md]: Added.

File: exemples/gt_non_recours/README.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+6. Installer les dépendances de `vizu.R` en entrant les instructions suivantes dans la `Console` :
+
+    ```R
+    install.packages('readr')
+    install.packages('dplyr')
+    install.packages('tidyr')
+    install.packages('ggplot2')
+    ```
+
+6. Exécuter `vizu.R` en sélectionnant tout le contenu du fichier puis en cliquant sur le bouton `Run` (sur la barre d'en-tête du fichier).
```
[the remark this answers is not in the document]
Comment (Anna-Livia) [reply exemples/gt_non_recours/README.md]: No 😅

File: exemples/gt_non_recours/README.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+* Le script `framework.py` exécute un ensemble de calculs de variables OpenFisca, d'après un ensemble de réformes sur les situations contenues dans `/situations`.  
+Les resultats sont repertoriés dans un fichier `csv`.
+
+* Le code `vizu.R` en langage [R](https://www.r-project.org) génère des courbes à partir des résultats `resultats-levels-.csv` du `framework.py`.
+
+
+### Installation & exécution de `framework.py`
+
+Le framework est développé en [python](https://www.python.org) et dépend de la version 2.7 de ce langage (cf. `runtime.txt`).
```
Comment (Anna-Livia) [inline exemples/gt_non_recours/README.md]: I don't understand the reference to `runtime.txt`
maybe:
```
type `python -V`in your terminal to check which version you are currently running
```

**Comment by Anna-Livia** [comment]
Great work ! One little change and we are good to go :)

**Pull request merged**